src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java

```diff
@@ -133,6 +134,19 @@ public void shouldProcessDependantResponseFromJson() throws Exception {
         assertThat(output).isEqualTo(expected);
     }
 
+    @Test
+    public void shouldProcessDefendantPartialResponseFromJson() throws Exception {
```

Comment:
I am not big fan of these tests as serialisation / deserialisation is tested implicitly by integration tests. I would consider removing that tests class completely. Open for discussion though.

Reply:
Yeah we did speak about this a couple of weeks ago, and I would be happy to remove it but we can see what others think before we make any decisions